dev_guide/templates.adoc

```diff
@@ -82,13 +121,27 @@ $ oc create -f <filename> -n <project>
 The template is now available to be selected for a configuration using the web
 console or the CLI.
 
-=== Generating a Configuration
-Generate a configuration with the following command:
+=== Generating a list of resources
+Generate a list of resources with the following command:
 
 ----
 $ oc process -f <filename>
 ----
 
+You can also process the uploaded template by calling the process command using:
```

Comment:
Try being consistent, once you're talking about generating and here about processing. I'm trying to think which one is better, but it should be used in both places, not to confuse users.